os/tools/mkbinheader.py

```diff
@@ -97,6 +97,11 @@
 # Temporary file to estimate the static RAM size.
 STATIC_RAM_ESTIMATION = 'temp_static_ram_estimation_file'
 
+if int(binary_stack_size) >= int(dynamic_ram_size) :
```

Comment:
@YiHyunJin What is the binary stack size? Isn't it the stack size of main function? I agree this change but it gives misunderstanding easily. Let's rename it.